### ISSUE TITLE
feat(svc): add dataset import endpoint

### ISSUE THREAD

**Issue by jsam** [SwissDataScienceCenter/renku-python#985]
We need a endpoint for importing external datasets via service to be used on the UI. The idea behind it is to expose the following endpoint:

```
POST /datasets/import

{
  "project_id": <project_id>,
  "dataset_uri": <dataset doi or http>,
  "short_name": "",
  "extract": True|False,
}
```

This endpoint will create and enqueue a job and return a job id to the requester. This newly created job needs to be visible in #984, since that endpoint can be used for long polling of the jobs.

The response example:

```
  "result": { "job_id": <> }
``` 

Prerequisites:  #984, #957

**Comment by jsam** [comment]
Potential improvements for this would be that we include exact progress of the import within jobs extras key (described in #984) which UI could use to display to the end user. I would have to see if core would support this in a seamless way, if it does we can do put it there already in v1, if not we should open a followup ticket for refactoring of the core to support this behavior.
